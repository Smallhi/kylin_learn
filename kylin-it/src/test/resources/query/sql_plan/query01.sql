SELECT ORDER_ID
FROM test_kylin_fact
GROUP BY CAST(CASE
		WHEN '1030101' = '1030101' THEN substring(COALESCE(LSTG_FORMAT_NAME, '888888888888'), 1, 1)
		WHEN '1030101' = '1030102' THEN substring(COALESCE(LSTG_FORMAT_NAME, '999999999999'), 1, 1)
		WHEN '1030101' = '1030103' THEN substring(COALESCE(LSTG_FORMAT_NAME, '777777777777'), 1, 1)
		WHEN '1030101' = '1030104' THEN substring(COALESCE(LSTG_FORMAT_NAME, '666666666666'), 1, 1)
	END AS varchar(256)), ORDER_ID