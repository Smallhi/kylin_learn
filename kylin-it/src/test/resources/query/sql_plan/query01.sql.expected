SELECT "ORDER_ID"
FROM "DEFAULT"."TEST_KYLIN_FACT"
GROUP BY SUBSTRING(CASE WHEN "LSTG_FORMAT_NAME" IS NOT NULL THEN "LSTG_FORMAT_NAME" ELSE '888888888888' END FROM 1 FOR 1), "ORDER_ID"